okhttp/src/main/java/com/squareup/okhttp/internal/spdy/SpdyStream.java

```diff
@@ -218,6 +231,12 @@ public void close(ErrorCode rstStatusCode) throws IOException {
     if (!closeInternal(rstStatusCode)) {
       return; // Already closed.
     }
+    // Close down dependant pushed streams
+    for(SpdyStream pushed : pushedStreams) {
+      int pushedId = pushed.getId();
+      pushed.getSource().close();
+      pushedStreams.remove(pushedId);
```

Comment:
It is fishy, but was in the original voxer branch for okhttp.
see: https://github.com/Voxer/okhttp/commit/f7f7f2b0e887d033d65dab61a0a2e6a7edb74097.
I'm also not sure if it really needs to do that or not.